traffic_ops/traffic_ops_golang/routing/routes.go

```diff
@@ -354,6 +354,11 @@ func Routes(d ServerData) ([]Route, []RawRoute, http.Handler, error) {
 		{1.3, http.MethodPut, `deliveryservices/{xmlID}/urisignkeys$`, urisigning.SaveDeliveryServiceURIKeysHandler, auth.PrivLevelAdmin, Authenticated, nil},
 		{1.3, http.MethodDelete, `deliveryservices/{xmlID}/urisignkeys$`, urisigning.RemoveDeliveryServiceURIKeysHandler, auth.PrivLevelAdmin, Authenticated, nil},
 
+		//Delivery Service Required Capabilities: CRUD
+		{1.4, http.MethodGet, `deliveryservice_required_capabilities/?$`, api.ReadHandler(&deliveryservice.RequiredCapability{}), auth.PrivLevelReadOnly, Authenticated, nil},
```

Comment:
this should be deliveryservices_required_capabilities (missing s on deliveryservice) perhttps://github.com/apache/trafficcontrol/blob/master/blueprints/server-capabilitites.md#traffic-ops-impact

Reply:
It seems like this should be singular as you are assigning a required capability to a single delivery service

Reply:
Whatever's decided you'll need to make sure the documentation matches.